operator/config/samples/ais_v1beta1_aistore.yaml

```
@@ -21,15 +21,16 @@ spec:
       - path: "/ais1"
         size: 10Gi
 
-  nodeImage: "aistore/aisnode:3.10"
+    # In certain environments (e.g. minikube), storage volumes attached to AIS targets may not have associated block devices.
+    # Alternatively, AIS targets may "see" multiple mountpath directories sharing a single given block device.
+    # In both of those cases, set allowSharedNoDisks = true (but note that this setting is **not recommended** for production).
+    allowSharedNoDisks: false
+
+
+  nodeImage: "aistore/aisnode:latest"
```

@dzhulgakov thanks for the PR. 

We are currently migrating our docker images from `aistore` to `aistorage`. Could you make this minor change? 
```suggestion
  nodeImage: "aistorage/aisnode:latest"
```